components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -58,6 +67,95 @@ class PlacesApi(path: String) : PlacesManager, AutoCloseable {
         private const val READ_WRITE: Int = 2
     }
 
+    @Throws(PlacesException::class) // ???
```

Comment:
I'm not sure `PlacesException` is correct here. I was using the logins component as a guide and I couldn't figure out where [LoginsStorageException](https://github.com/mozilla/application-services/blob/574450d9f3c1f5c9c6c2a71345cf0678440e985f/components/logins/android/src/main/java/mozilla/appservices/logins/DatabaseLoginsStorage.kt#L34) is declared so for now `PlacesException` is a placeholder for perhaps `PlacesError` or some other error type.